onnxruntime/test/python/quantization/test_op_argmax.py

```diff
@@ -105,6 +106,17 @@ def quantize_argmax_test(self, activation_type, weight_type, extra_options = {})
         data_reader.rewind()
         check_model_correctness(self, model_fp32_path, model_uint8_qdq_path, data_reader.get_next())
 
+        # Verify QDQ mode for TensorRT
```

Comment:
What do you want to check for TensorRT here?

Reply:
I want to make sure the tool insert QDQ before ArgMax even we don't quantize the output of Conv. Please tell me if the test doesn't verify what I mentioned/adding the test in wrong place/this test doesn't improve the code coverage.

Reply:
'extra_options = {'OpTypesToExcludeOutputQuantizatioin': ['Conv']})' only excludes to quantize the output of Conv when the output is an output of graph. You can use option 'op_types_to_quantize=['ArgMax']' to only quantize ArgMax

Reply:
Fixed.